ar_mysql_flexmaster.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("rake")
   gem.add_development_dependency("wwtd")
   gem.add_development_dependency("minitest")
+  gem.add_development_dependency("minitest-reporters")
```

Comment:
use maxitest + `require 'maxitest/autorun'` ... solves a bunch of other minitest shortcomings too

Reply:
maxitest just shows green dots. What I want is

> When one test causes the following tests to fail, it is mighty practical to have the test names output as we go. Dots don't cut it.